pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -532,10 +532,9 @@ cpdef inline object parse_timedelta_unit(object unit):
     ----------
     unit : an unit string
     """
-    if unit is None:
-        return 'ns'
-    elif unit == 'M':
+    if unit is None or unit == 'M':
         return unit
```

Comment:
@jbrockmendel just explaining some changes to previous code so you can review, here I am leaving unit untouched if `None` because I thought was better only to do "nanoseconds if `None`" in the functions in the very end of the chain (where it is impossible to proceed without having something for units), not in the intermediate ones like this one here. This is because `None` is a perfectly valid state (user did not provide units) and we want to preserve it as long as possible to ensure correct behaviour through the whole process.

Reply:
yes this is ok, can you add a comment